### ISSUE TITLE
Error while installing pods

### ISSUE THREAD

**Issue by allanmacatingrao** [CocoaPods/CocoaPods#7377]
### Command

```
/usr/local/bin/pod install
```

### Report

* What did you do?
I added a new developer in my bitbucket account under specific repo,
when he cloned the repo, run pod install, it generated the error below.

* What did you expect to happen?
at least all pods will be installed without error

* What happened instead?
Error below

### Stack

```
   CocoaPods : 1.4.0
        Ruby : ruby 2.3.3p222 (2016-11-21 revision 56859) [universal.x86_64-darwin17]
    RubyGems : 2.5.2
        Host : Mac OS X 10.13.2 (17C205)
       Xcode : 7.3.1 (7D1014)
         Git : git version 2.7.4 (Apple Git-66)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 6218c16db238fa19c829798183dd605f05a60fe4
```

### Plugins

```
cocoapods-clean       : 0.0.1
cocoapods-deintegrate : 1.0.2
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.0
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project

platform :ios, '8.0'
use_frameworks!

def shared_pods
    # 87.27
    pod 'AFNetworking', '~> 2.5.0'
    
    # working
    pod 'SDWebImage'
    
    # pod 'SWTableViewCell'
    pod 'FastImageCache'
    
    # unused
    # pod 'ABPadLockScreen', '~> 3.3.0'
    
    pod 'EDStarRating'
    
    pod 'KeychainAccess', '~> 2.4.0'
    pod 'SWXMLHash', '~> 2.1.0'
    pod 'SSZipArchive'
    
    # custom
    pod 'MBProgressHUD-Imonggo', { git: 'https://github.com/allanmacatingrao/MBProgressHUD-Imonggo.git' }
    pod 'NYAlertViewController-Imonggo', { git: 'https://github.com/allanmacatingrao/NYAlertViewController-Imonggo.git' }
    pod 'KAProgressLabel-Imonggo', { git: 'https://github.com/allanmacatingrao/KAProgressLabel-Imonggo.git' }
    pod 'NumPad-Imonggo', { git: 'https://github.com/allanmacatingrao/NumPad-Imonggo.git' }
    
    pod 'sReto', '~> 1.4.1'
    
    
    
    # image upload
    # pod 'TGCameraViewController' <<< Not working
    
    pod 'TGCameraViewController', :path => './'
    pod 'TOCropViewController', '~>2.0.11'
end

target 'Imonggo_Store' do

   # 87.27
   pod 'AFNetworking', '~> 2.5.0'
	
	# working
	pod 'SDWebImage'
	
	# pod 'SWTableViewCell'
	pod 'FastImageCache'
	
	# unused
    # pod 'ABPadLockScreen', '~> 3.3.0'
	
	pod 'EDStarRating'
	
	pod 'KeychainAccess', '~> 2.4.0'
	pod 'SWXMLHash', '~> 2.1.0'
	pod 'SSZipArchive'
	
	# custom
	pod 'MBProgressHUD-Imonggo', { git: 'https://github.com/allanmacatingrao/MBProgressHUD-Imonggo.git' }
	pod 'NYAlertViewController-Imonggo', { git: 'https://github.com/allanmacatingrao/NYAlertViewController-Imonggo.git' }
	pod 'KAProgressLabel-Imonggo', { git: 'https://github.com/allanmacatingrao/KAProgressLabel-Imonggo.git' }
	pod 'NumPad-Imonggo', { git: 'https://github.com/allanmacatingrao/NumPad-Imonggo.git' }
	
	pod 'sReto', '~> 1.4.1'
	
	
	# image upload
	# pod 'TGCameraViewController' <<< Not working
	
	pod 'TGCameraViewController', :path => './'
	pod 'TOCropViewController', '~>2.0.11'
    shared_pods
	
	
	pod 'Fabric'
	pod 'Crashlytics'
end

#target 'ImonggoJFCUITests' do
#	pod 'KIF', '~> 3.2.0'
#end
#target 'Imonggo iPad JFCTests' do
#	pod 'KIF', '~> 3.2.0'
#end
```

### Error

```
Errno::ENOENT - No such file or directory @ realpath_rec - /Users/eleazertoluan/Documents/imonggo-enterprise-jfc/LICENSE
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/project.rb:190:in `realpath'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/project.rb:190:in `realpath'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/project.rb:190:in `add_file_reference'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:122:in `block (3 levels) in add_developer_files'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:120:in `each'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:120:in `block (2 levels) in add_developer_files'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:115:in `each'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:115:in `block in add_developer_files'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/user_interface.rb:142:in `message'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:114:in `add_developer_files'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb:43:in `install!'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator.rb:163:in `install_file_references'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer/xcode/pods_project_generator.rb:64:in `generate!'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer.rb:184:in `block in generate_pods_project'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/user_interface.rb:64:in `section'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer.rb:183:in `generate_pods_project'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/installer.rb:119:in `install!'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/command/install.rb:41:in `run'
/Library/Ruby/Gems/2.3.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.3.0/gems/cocoapods-1.4.0/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:22:in `load'
/usr/local/bin/pod:22:in `<main>'
```

**Comment by allanmacatingrao** [comment]
How do i move forward from this?

**Comment by dnkoutso** [comment]
@allanmacatingrao if you control the pod then fix the podspec to point to the right `LICENSE` file...in the meantime you can either do the following:

1) Fork the podspec that fixes the `LICENSE` missing file and publish to your own repo.
2) Use CocoaPods 1.3.1 until this is fixed.
3) Use CocoaPods 1.4.0 but monkey patch the method `add_developer_files` to do nothing
4) Wait for a fix that should come hopefully this coming week and use the master version of pods which is basically 1.4.0

**Comment by allanmacatingrao** [comment]
Thank you. Just did #2.

**Comment by amorde** [comment]
Sure thing - will get a PR out as soon as I can

**Comment by dnkoutso** [comment]
Just to be certain this might occur on any developer file so perhaps not only just `LICENSE`. I also do not remember if my claim for `--allow-warnings` is 100% valid around `LICENSE` files....it might be an error and therefore its impossible for an author to publish. Will double check this week.

**Comment by qq912276337** [comment]
use CocoaPods 1.3.1 can do it

**Comment by dnkoutso** [comment]
this has been fixed in for 1.5.0
